documentation/src/main/asciidoc/ch12.asciidoc

```diff
@@ -456,7 +456,8 @@ custom extensions for both of these interfaces.
 `HibernateConstraintValidatorContext` is a subtype of `ConstraintValidatorContext` which allows you to:
 
 * set arbitrary parameters for interpolation via the Expression Language message interpolation
-facility using `HibernateConstraintValidatorContext#addExpressionVariable(String, Object)`.
+facility using `HibernateConstraintValidatorContext#addExpressionVariable(String, Object)`
+or `HibernateConstraintValidatorContext#addMessageParameter(String, Object)`.
```

Comment:
I didn't check the example, but do you explain somewhere what the difference between those two methods is?

Reply:
Not really, I added more information.